website/docs/cluster-management/managing-existing-clusters.mdx

```diff
@@ -36,146 +36,154 @@ kubectl create secret generic demo-01-kubeconfig \
 
 1. Create a new service account on the remote cluster:
 
-```yaml
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: demo-01
-  namespace: default
-```
+	```yaml
+	apiVersion: v1
+	kind: ServiceAccount
+	metadata:
+		name: demo-01
+		namespace: default
+	```
+
+1. Add RBAC permissions for the service account
+
+	<details><summary>Expand to see role manifests</summary>
+	```yaml
+	---
+	apiVersion: rbac.authorization.k8s.io/v1
+	kind: ClusterRoleBinding
+	metadata:
+		name: impersonate-user-groups
+	subjects:
+		- kind: ServiceAccount
+			name: demo-01
+			namespace: default
+	roleRef:
+		kind: ClusterRole
+		name: user-groups-impersonator
+		apiGroup: rbac.authorization.k8s.io
+	---
+	apiVersion: rbac.authorization.k8s.io/v1
+	kind: ClusterRole
+	metadata:
+		name: user-groups-impersonator
+	rules:
+		- apiGroups: [""]
+			resources: ["users", "groups"]
+			verbs: ["impersonate"]
+		- apiGroups: [""]
+			resources: ["namespaces"]
+			verbs: ["get", "list"]
+	```
+
+	</details>
+
+	This will allow WGE to introspect the cluster for available namespaces.
+
+	Once we know what namespaces are available we can test whether the logged in user can access them via impersonation.
+
+1. Get the token of the service account
+
+	First get the list of secrets of the service accounts by running the following command:
+
+	```sh
+	kubectl get secrets --field-selector type=kubernetes.io/service-account-token
+	NAME                      TYPE                                  DATA   AGE
+	default-token-lsjz4       kubernetes.io/service-account-token   3      13d
+	demo-01-token-gqz7p       kubernetes.io/service-account-token   3      99m
+	```
+
+	`demo-01-token-gqz7p` is the secret that holds the token for `demo-01` service account
+
+	To get the token of the service account run the following command:
+
+	```sh
+	TOKEN=$(kubectl get secret demo-01-token-gqz7p -o jsonpath={.data.token} | base64 -d)
+	```
+
+1. Create a kubeconfig secret
+
+	We'll use a helper script to generate the kubeconfig, save this into `static-kubeconfig.sh`:
+
+	<details><summary>Expand to see script</summary>
```

Comment:
This looks good 👍 
<img width="1228" alt="Screenshot 2023-02-21 at 10 34 59" src="https://user-images.githubusercontent.com/12957664/220321561-fcfaea51-cda2-40c9-8f05-520a2afdf3d5.png">